src/nvhttp.cpp

```diff
@@ -842,6 +830,7 @@ namespace nvhttp {
       if (video::probe_encoders()) {
         tree.put("root.resume", 0);
         tree.put("root.<xmlattr>.status_code", 503);
+        tree.put("root.<xmlattr>.status_message", "Failed to initialize video capture/encoding. Is a display connected and turned on?");
```

Comment:
```suggestion
        tree.put("root.<xmlattr>.status_message", "Failed to initialize video capture/encoding. Is a display connected to the selected adapter, and turned on?");
```

This might make it too wordy, but I think a lot of users don't understand the display needs to be connected to the GPU.

Reply:
Most users probably won't be selecting adapters (and I agree it might be too wordy). I'll put a message next to the encoder and adapter selection fields in the UI instead.